sdk/python/lib/pulumi/output.py

```diff
@@ -37,6 +39,7 @@
 Inputs = Mapping[str, Input[Any]]
 
 
+@known_types.output
```

Comment:
Definitely would be good to get @CyrusNajmabadi's eyes on the output/runtime stuff.